angular/src/app/services/wallet-manager.ts

```diff
@@ -1,36 +1,37 @@
 // import { HDKey } from "micro-bip32"; // TODO: Uninstall the previous package, replaced with @scure.
-import { HDKey } from '@scure/bip32';
-import { mnemonicToSeedSync } from '@scure/bip39';
+import {HDKey} from '@scure/bip32';
```

Comment:
Here your editor is clearly not formatting like VS Code =)

Reply:
My bad.. will look into it